rpc_jobs/build_summary.yml

```diff
@@ -44,7 +29,7 @@
     parameters:
       - string:
           name: "DOCKER_IMAGE"
-          default: "rcbops/buildsummary"
+          default: "buildsummary"
           description: |
             Hours. Instances older than this will be removed.
       - rpc_gating_params
```

Comment:
[L54](https://github.com/rcbops/rpc-gating/pull/1293/files#diff-9d1caf1170f2e050a035c7ff42bdd282R54) below might need to be wrapped in withinternalregistry?